AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java

```diff
@@ -0,0 +1,9 @@
+package com.tap.project.escrivaghera.AccountantApp;
+
+public class IllegalJournalEntryException extends Exception {
+
+	public IllegalJournalEntryException(String error) {
```

Comment:
the constructor is empty....the string is passed from the class who call the exception. Look at the class who call that exception.